### DISCUSSION TITLE
Promoting Dataset Reuse Via a Dataset Usability Score

### DISCUSSION THREAD

**Discussion by jrich71** [NIH-GREI/grei-pm#168] [Ideas]
### If applicable, select the GREI Year 3 Objective, Task, or Deliverable.

- [X] Community Engagement (All units)
- [X] Desirable Repository Characteristics
- [ ] Consistent Metadata
- [ ] Search & Browse
- [ ] Analytics & Reporting
- [X] Use Cases
- [ ] Connecting Digital Objects
- [ ] QA/QC
- [ ] Training & Community Engagement

### Please share your idea or use case.

```bash
Kaggle provides a dataset usability score to inform potential users about the LOE necessary for working with a dataset. https://www.kaggle.com/discussions/product-feedback/93922 Might not future GRs also promote a standard dataset usability score to help researchers initially gauge the LOE necessary to work with, and build from, datasets within a GR?

More broadly, GREI could engage Kaggle (or similar sites) for lessons learned regarding data reuse that we could consider for ourselves.
```

**Comment by cmbz** [comment]
Hi @jrich71 Relatedly, [Dataverse 6.4 now supports Croissant metadata export](https://github.com/IQSS/dataverse/releases). Additional information about the exporter here: [Croissant Exporter for Dataverse](https://github.com/gdcc/exporter-croissant) and [here](https://groups.google.com/g/dataverse-community/c/JI8HPgGarr8/m/DqEIkiwlAgAJ).
